tests/mock_utils.py

```diff
@@ -7,6 +7,7 @@
 TEST_PROCESS_FLAT_ARN = "arn:aws:sns:us-east-1:123456789012:test-flat"
 TEST_PROCESS_STUDY_META_ARN = "arn:aws:sns:us-east-1:123456789012:test-meta"
 TEST_CACHE_API_ARN = "arn:aws:sns:us-east-1:123456789012:test-cache"
+TEST_LOCKFILE_CLEANUP_URL = "https://sqs.us-east-1.amazonaws.com/123456789012/test-lockfile-cleanup"
```

Comment:
these services are 95% the same thing but have completely different APIs/naming conventions. 🤷